### PR TITLE
MF-942 - Fix email template logic 

### LINKED ISSUE #942: E-mail template usage is confusing
**BUG REPORT**

1. What were you trying to achieve?

Use e-mail template, change path, make is usable on Windows Server

2. What are the expected results?

E-mail template usage should be clear to se from the code (seld documented)

3. What are the received results?

Code is confusing and not self-documenting:
- `docker-compose` uses: https://github.com/mainflux/mainflux/blob/841b9437184015415181c23f689f2cc38e33b8ef/docker/docker-compose.yml#L70 and not the one from the [configs](https://github.com/mainflux/mainflux/tree/master/configs) - even more, these two are different.
- What is the purpose than of one in the configs?
- ENV variable in `.env` is set like [this](https://github.com/mainflux/mainflux/blob/841b9437184015415181c23f689f2cc38e33b8ef/.env#L4- In the [default value](https://github.com/mainflux/mainflux/blob/841b9437184015415181c23f689f2cc38e33b8ef/cmd/users/main.go#L68) it is again set differently, and even more confusing with `../<something>`. Having in mind that in containers binaries are put into root, this `..` does not make sense.

Can we use just filename here and let user map this file wherever he wants?

4. What are the steps to reproduce the issue?

5. In what environment did you encounter the issue?

6. Additional information you deem important:

### PR DESCRIPTION
**Pull request by mteodor**
### Which issue(s) does this PR fix/relate to?
Resolves #942

### Did you document any new/modified functionality?
Yes

### CODE REVIEW

**Review by blokovi — DISMISSED** [review]

File: users/README.md [hunk 1]
```diff
@@ -41,9 +41,9 @@ default values.
 | MF_EMAIL_PASSWORD         | Mail server password                                                    |                |
 | MF_EMAIL_FROM_ADDRESS     | Email "from" address                                                    |                |
 | MF_EMAIL_FROM_NAME        | Email "from" name                                                       |                |
-| MF_TOKEN_SECRET           | Token secret for signing token used for password reset                  |                |
+| MF_EMAIL_TEMPLATE         | Email template for sending emails with password reset link              | email.tmpl     |
+| MF_TOKEN_SECRET           | Password reset token signing secret                    |                |
```
Comment (drasko) [inline users/README.md]: Align this properly

File: users/README.md [hunk 2]
```diff
@@ -105,9 +105,11 @@ make users
...
 ```
 
+if 'MF_EMAIL_TEMPLATE' doesn't point to any file service will function but password reset functionality will not work
```
Comment (drasko) [inline users/README.md]: Sentences start with capital letter.

Use backticks (``) rather than single qoutes.

**Review by drasko — CHANGES_REQUESTED** [review]

**Review by manuio — APPROVED** [review]

**Review by anovakovic01 — APPROVED** [review]
LGTM!

**Review by nmarcetic — APPROVED** [review]

**Review by manuio — APPROVED** [review]
LGTM

**Review by drasko — APPROVED** [review]
LGTM

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**